stream_alert/sink.py

```diff
@@ -81,6 +84,66 @@ def _get_sns_topic_arn(self):
             topic=topic
         )
 
+    def _chunk_sns_messages(self):
+        """Return a list of base64 encoded SNS messages.
+
+        These are split to ensure each is less than 256kb in size.
```

Comment:
This code comment is a bit confusing, what is "these" and what does "split" mean in this context. Clarify